### ISSUE TITLE
Subquery cannot preload associations

### ISSUE THREAD

**Issue by nicholasjhenry** [mojotech/scrivener_ecto#19]
@drewolson just a heads up this commit broke queries with `preload`: c6d1c45f3b4adb9014cbbb88ed4becd2bf6ad0e8

Subqueries cannot contain preloads, so you would still need to remove it, if you wanted to use the subquery syntax. So in the example from the README:

```elixir
  page =
    MyApp.Person
    |> where([p], p.age > 30)
    |> order_by(desc: :age)
    |> preload(:friends)
    |> MyApp.Repo.paginate(params)
```

Since we have `preload(:friends)` in the query (which is used in a subquery for pagination) this error will be raised `(Ecto.QueryError) cannot preload associations in subquery in query`

**Comment by drewolson** [comment]
Thanks, I'm pushing a fix for this as we speak. Will update this issue.

**Comment by drewolson** [comment]
This is now fixed in `1.1.3` and released.
